src/components/definition-list/dd.component.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { StyledDd } from './definition-list.style';
+
+const Dd = ({
+  mb = 2,
+  children
+}) => {
+  return (
+    <StyledDd
+      mb={ mb }
+    >
+      {children}
+    </StyledDd>
+  );
+};
+
+Dd.propTypes = {
+  /** Content margin bottom. This value will be multiplied by the theme spacing constant (8) */
+  mb: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6]),
+  /** Prop to render string or node in the `<Dd></Dd>` tags */
+  children: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired
```

Comment:
This can just be `children: PropTypes.node.isRequired`  as nothing special is happening with strings here, you would just pass your text in the normal HTML way, i.e. `<Dd>Some text</Dd>`.

On the prop-types docs you can see that `PropTypes.node` covers strings anyway https://www.npmjs.com/package/prop-types

Reply:
Unless they can only be strings within a definition list, in which case you should only allow strings

Reply:
Wow. Learn something new everyday. I'll get that changed.

Reply:
✅